lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -261,7 +272,7 @@ def get_comment_list_url(self, obj, endorsed=None):
         ):
             return None
         path = reverse("comment-list")
-        query_dict = {"thread_id": obj["id"]}
+        query_dict = OrderedDict(thread_id=obj["id"])
```

Comment:
I'm curious as to why this change is required, does the order matter here?

Reply:
Since I picked up this PR from Aayush I built on his changes. I don't know the context for this change, honestly doesn't make much sense to me, but I forgot to check if it's necessary. Will check and remove if unneeded.